example/package.json

```diff
@@ -12,19 +12,19 @@
     "test": "jest"
   },
   "dependencies": {
-    "expo": "^40.0.0",
-    "expo-splash-screen": "~0.8.1",
-    "react": "16.13.1",
-    "react-dom": "16.13.1",
-    "react-native": "0.63.4",
-    "react-native-unimodules": "~0.12.0",
-    "react-native-web": "~0.14.9"
+    "expo": "^43.0.0-beta.3",
```

Comment:
shoudn't we eject it? 🤔

Reply:
no, why? expo is needed to open example with react-native-select-pro and this is package.json from example folder. in root package.json of course we have not expo